bindings/nodejs/lib/types/models/api/plugins/indexer/IOutputsResponse.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type { HexEncodedString } from '../../../../utils/hexEncodedTypes';
+/**
+ * Details of an outputs response from the indexer plugin.
+ */
+export interface IOutputsResponse {
```

Comment:
There is class `export class OutputIdsResponse {` as well, we only need one of them?